Help/js/help.js

```diff
@@ -1,3 +1,4 @@
+/* global ls: false, privlyNetworkService: false*/
```

Comment:
I think you can put this below the @fileoverview? The file overview should be the first comment.